handlers/get_binding.go

```diff
@@ -61,5 +61,6 @@ func (h APIHandler) GetBinding(w http.ResponseWriter, req *http.Request) {
 			VolumeMounts:    binding.VolumeMounts,
 		},
 		Parameters: binding.Parameters,
+		Endpoints:  binding.Endpoints,
```

Comment:
@TheSp1der Can you please move this to the corresponding struct above?

Reply:
My apologies, I should have run this through my own tests. I have submitted another commit. Please take your time reviewing it, I also made a change to `handlers/bind.go` to support the response there as well as you pointed out it it should be returned for that as well.